extensions/aria_extension_tosca/simple_v1_0/modeling/__init__.py

```diff
@@ -446,6 +451,29 @@ def get(name):
     return model
 
 
+def create_workflow_operation_template_model(context, service_template, policy):
+    model = OperationTemplate(name=policy._name,
```

Comment:
You are not wrong ... but I also wanted to minimize the number of models we have. I think if we do decide to change this, we can do it in a separate JIRA.